.buildkite/scripts/steps/package.sh

```diff
@@ -13,8 +13,13 @@ fi
 export AGENT_DROP_PATH=build/elastic-agent-drop
 mkdir -p $AGENT_DROP_PATH
 
+if [ "$FIPS" = "true" ]; then
+# Download the components from the MANIFEST_URL and then package those downloaded into the $AGENT_DROP_PATH
+mage clean downloadManifest packageUsingDRA fixDRADockerArtifacts
+else
 # Download the components from the MANIFEST_URL and then package those downloaded into the $AGENT_DROP_PATH
 mage clean downloadManifest packageUsingDRA ironbank fixDRADockerArtifacts
+fi
```

Comment:
How about we explore using something that is less commonly duplicated?

```suggestion
mageIronbankCommand=
if [ "$FIPS" = "true" ]; then
  mageIronbankCommand=ironbank
fi
echo "Download the components from the MANIFEST_URL and then package those downloaded into the $AGENT_DROP_PATH"
mage clean downloadManifest packageUsingDRA $mageIronbankCommand fixDRADockerArtifacts
```

I decided to remove the `comment` for an `echo`. The above proposal removes the duplicated mage goals

Reply:
We could use an array:

```suggestion
mageOptions=(clean downloadManifest packageUsingDRA fixDRADockerArtifacts)

# Stop building ironbank for FIPS
if [ "$FIPS" != "true" ]; then
    mageOptions+=("ironbank")
fi

# Download the components from the MANIFEST_URL and then package those downloaded into the $AGENT_DROP_PATH
mage "${mageOptions[@]}"
```

Reply:
@v1v  for the moment I went with the mage target array in the package.sh script (your second suggestion).
We can always refactor in a follow-up PR, feel free to open an issue with your proposal so we can discuss on it.